### PR TITLE
feat: added scss files so each itcss layer can be imported individually

### PR DESCRIPTION
**Pull request by danbull**
re #189

**Description**
Currently any project consuming Gravity has to include gravity scss or compiled css first and then include local, project specific css afterwards. This new way of importing scss files allows projects to consume each ITCSS layer individually and interlace project specific scss. This means the entire project can follow the ITCSS architecture.

**Checklist**
### Other
- [ ] Documentation for any new feature and functionality has been added.

### CODE REVIEW

**Comment by danbull** [comment]
@james-nash @dw-buildit let me know what you think, if we're happy with the structure I'll add some docs to #189

File: src/styleguide/_patterns/_generated/symbols.mustache [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<svg style="display: none;" aria-hidden="true" xmlns="http://www.w3.org/2000/svg">
```
Comment (james-nash) [inline src/styleguide/_patterns/_generated/symbols.mustache]: Oh. This file shouldn't be checked into git.

It gets generated by the Pattern Lab build. However, as part of the recent Nunjucks conversion, the `.gitignore` rules was also updated. This is probably a left over from builds you did while we still used Mustache.

Please remove it.

**Comment by james-nash** [comment]
@danbull Btw, there's no need to update the CHANGELOG, `semantic-release` does that for us now.

**Comment by danbull** [comment]
> Looks good to me. Thanks for doing this!
> 
> One comment though: Is the `_settings.all.scss` naming not a tad verbose? Why not just `_settings.scss` and so on?
> 
> That being said, I don't feel strongly about that, so approving this PR. Keep the names as they are or change them before merging - I'll leave it up to you. :-)

Yeah I had that originally but then saw Harry Roberts used `.all` and thought it might be useful - in the future we may have things like `components.forms.scss`, for example

**Pull request merged**